rpcd/etc/openstack_deploy/user_variables.yml

```diff
@@ -89,3 +89,13 @@ tempest_service_available_ceilometer: False
 neutron_neutron_conf_overrides:
   nova:
     endpoint_type: internal
+
+# Disable glance image cache and reconfigure cinder conversion dir outside of a small root partition
+glance_flavor: keystone
+glance_glance_api_conf_overrides:
+  DEFAULT:
+    image_cache_dir: None
+
+cinder_cinder_conf_overrides:
+  DEFAULT:
+    image_conversion_dir: '/openstack/backup'
```

Comment:
'/openstack/backup' seems like an inappropriate directory. OSA appears to be using this as a container backup.